infinity/formats/itm_v1.py

```diff
@@ -485,7 +628,7 @@ class ITM_V10_Format (Format):
             { 'key': 'opcode_number',
               'type': 'WORD',
               'off': 0x0000,
-              'enum': 'effects',
+              'enum': 'effects', # FIXME: this ids file does not exist
               'label': 'Opcode number'},
```

Comment:
I didn't think of that. How can it be used outside of gemrb environment? I'd still add enums for game types other than gemrb.